packages/forms-web-app/src/controllers/full-appeal/submit-appeal/enter-code.js

```diff
@@ -14,6 +14,10 @@ const getEnterCode = async (req, res) => {
 	const {
 		session: { appeal }
 	} = req;
+
+	if (!appeal || !appeal.state || appeal.state === 'SUBMITTED') {
```

Comment:
this should be `if ( appeal && appeal.state && appeal.state === 'SUBMITTED)`

Reply:
this wouldn't work, per comment below